src/overlays/misc/ovl_kaleido_scope/z_kaleido_scope_PAL.c

```diff
@@ -553,55 +553,55 @@ void KaleidoScope_DrawPages(PlayState* play, GraphicsContext* gfxCtx) {
             }
 
             if (pauseCtx->stickAdjX < -30) {
-                if (D_8082AD4C == -1) {
-                    if (--D_8082AD44 < 0) {
-                        D_8082AD44 = XREG(6);
+                if (sStickXRepeatState == -1) {
+                    if (--sStickXRepeatTimer < 0) {
+                        sStickXRepeatTimer = R_PAUSE_STICK_REPEAT_DELAY_FIRST;
                     } else {
                         pauseCtx->stickAdjX = 0;
                     }
                 } else {
-                    D_8082AD44 = XREG(8);
-                    D_8082AD4C = -1;
+                    sStickXRepeatTimer = R_PAUSE_STICK_REPEAT_DELAY;
+                    sStickXRepeatState = -1;
                 }
             } else if (pauseCtx->stickAdjX > 30) {
-                if (D_8082AD4C == 1) {
-                    if (--D_8082AD44 < 0) {
-                        D_8082AD44 = XREG(6);
+                if (sStickXRepeatState == 1) {
+                    if (--sStickXRepeatTimer < 0) {
+                        sStickXRepeatTimer = R_PAUSE_STICK_REPEAT_DELAY_FIRST;
                     } else {
                         pauseCtx->stickAdjX = 0;
                     }
                 } else {
-                    D_8082AD44 = XREG(8);
-                    D_8082AD4C = 1;
+                    sStickXRepeatTimer = R_PAUSE_STICK_REPEAT_DELAY;
+                    sStickXRepeatState = 1;
```

Comment:
I think you have `R_PAUSE_STICK_REPEAT_DELAY_FIRST` and `R_PAUSE_STICK_REPEAT_DELAY` swapped. 

The first repeated input is 10 frames delayed, then all subsequent repeated inputs are 2 frames delayed.

Reply:
Good catch, how did I mess that up NotLikeThis